services/QuillLMS/client/app/bundles/Evidence/components/studentView/stepOverview.tsx

```diff
@@ -1,43 +1,63 @@
 import * as React from 'react'
 
-import { greenCheckIcon, encircledWhiteArrowIcon, } from '../../../Shared/index'
-
-const READ_PASSAGE_STEP = 1
+import { whiteCheckGreenBackgroundIcon, READ_PASSAGE_STEP, BECAUSE_PASSAGE_STEP, BUT_PASSAGE_STEP, SO_PASSAGE_STEP } from '../../../Shared/index'
 
 const steps = [
   {
-    text: 'Read a text and highlight sentences'
+    html: <p>Read a text and highlight sentences.</p>,
+    stepNumber: READ_PASSAGE_STEP
+  },
+  {
+    html: <p>Expand a sentence using <u>because</u> to provide a reason.</p>,
+    stepNumber: BECAUSE_PASSAGE_STEP
+  },
+  {
+    html: <p>Expand a sentence using <u>but</u> to provide an opposing idea.</p>,
+    stepNumber: BUT_PASSAGE_STEP
   },
   {
-    text: 'Write three sentences using evidence'
+    html: <p>Expand a sentence using <u>so</u> to provide a result.</p>,
+    stepNumber: SO_PASSAGE_STEP
   }
 ]
```

Comment:
If you just define these individually here (so like:

```const readPassageStep = {
    html: <p>Read a text and highlight sentences.</p>,
    stepNumber: READ_PASSAGE_STEP
}```

You can just call those individually when you pass them as the `step` prop to the `Step` component throughout the file and then you don't need to mess with the stepNumber - 1 thing. It doesn't look like you actually need them as an array at all.